var/spack/repos/builtin/packages/libjpeg-turbo/package.py

```diff
@@ -34,7 +34,9 @@ class LibjpegTurbo(AutotoolsPackage):
     homepage = "http://libjpeg-turbo.virtualgl.org"
     url      = "http://downloads.sourceforge.net/libjpeg-turbo/libjpeg-turbo-1.3.1.tar.gz"
 
-    version('1.5.0', '3fc5d9b6a8bce96161659ae7a9939257')
+    version('1.5.1', '55deb139b0cac3c8200b75d485fc13f3')
+    version('1.5.0', '3fc5d9b6a8bce96161659ae7a9939257', preferred=True)
```

Comment:
Best practice is to add the new version and only use `preferred=True` in exceptional circumstances.  (For example, it is used to prefer Python 2.7 over Python 3 by default; or when a new version is buggy and should be avoided in most cases).  The rebuild issue is annoying, but not to be solved in this way.  It will be addressed with a new concretizer and work on Spack Environments.